### ISSUE TITLE
For first-time issue filers, bot notifies about WHATWG workmode and process

### ISSUE THREAD

**Issue by travisleithead** [whatwg/meta#114]
Hi WHATWG!

(was considering titling this: "Proposal: replace @annevk with a bot" :-)

Noted in several issues that 1st-time issue filers tend to get a courtesy note about the WHATWG working mode and policies. E.g.:
* https://github.com/whatwg/html/issues/3611#issuecomment-378863084
* https://github.com/whatwg/html/issues/3627#issuecomment-381405815

I think these notices are good. These courtesy notices help ensure newbies are aware of the process. Also noticed that the message has to be carefully crafted so as to not be percieved as a knock against the idea/issue. Hence having a bot deliever the message is probably good for PR.

(and yes, this is my first time filing an issue in this repo...)

**Comment by annevk** [comment]
I thought about this while doing it a few times and I might have even discussed it, but I think I like the personal touch. It only takes a couple seconds and a human has to read the issue anyway.

**Comment by domenic** [comment]
We could also try to make the CONTRIBUTING.md links contain some more of this information, since GitHub automatically links to those. E.g. https://github.com/whatwg/html/blob/master/CONTRIBUTING.md is rather focused on the mechanics of pull request contributions, and not on "how should I approach this community when filing issues".

**Comment by jfbrennan** [comment]
I created https://github.com/whatwg/html/issues/3627 (the little `autoselect` proposal) and I liked having someone reach out like @annevk  did. 

The comment did give me the impression that I got something wrong, but later understood it was just a courtesy/reminder. Maybe something like, "Thank you, and in an effort to ensure all requests meet expectations we ask all new filers to read the following guidelines..." would make it clear that the person isn't replying because something is wrong.
